### PR TITLE
Fix Note

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8595

### CODE REVIEW

**Comment by scanum** [comment]
[@JohanFreelancer9](https://github.com/JohanFreelancer9) Copy editing is needed for this PR. Thanks.

File: Teams/rooms/enroll-a-device.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ The Enrollment process involves these steps:
    ![Screenshot of settings and self-enrollment keys.](../media/software-installation-005new.png)
  
 > [!NOTE]
-In case you need to install the agent without the Teams App on the MTR being able to login to Teams. You can use our enrollment key as an optional process.Go to  '?'  (Help) on the top right corner of the portal, then select 'Download Installer' when installing the agent Place the ‘Self-Enrollment key’ (previously downloaded from the portal) on the **C:\Rigel** directory of the device.
+> In case you need to install the agent without the Teams App on the MTR being able to login to Teams. You can use our enrollment key as an optional process. Go to  '?'  (Help) on the top right corner of the portal, then select 'Download Installer' when installing the agent Place the ‘Self-Enrollment key’ (previously downloaded from the portal) on the **C:\Rigel** directory of the device.
```
Comment (JohanFreelancer9) [inline Teams/rooms/enroll-a-device.md:120]:
```suggestion
> If you need to install the agent without the Teams App on the MTR being able to log in to Teams, you can use our enrollment key as an optional process. Go to **?** (Help) on the top right corner of the portal, then select **Download Installer**. When installing the agent, place the **Self-Enrollment** key (previously downloaded from the portal) on the **C:\Rigel** directory of the device.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by get-itips** [comment]
closing to create a new pr as this now has conflicts

**Pull request closed without merging**